### PR TITLE
docs: blackflag.dev added to demo pages on docs site

### PR DESCRIPTION
**Pull request by openblackflag**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Added website https://www.blackflag.dev to docs site demo page. The site should have a performance score of 100 according to Pagespeed metrics.

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x ] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by zanettin** [comment]
great scores 🎉 
https://pagespeed.web.dev/report?url=https%3A%2F%2Fwww.blackflag.dev%2F&form_factor=mobile

**Comment by shairez** [comment]
nice!
merging

File: packages/docs/scripts/pages.json [hunk 1]
```diff
@@ -70,6 +70,11 @@
     "href": "https://sanyamjainqwik.vercel.app/",
     "tags": "portfolio"
   },
+  {
+    "href": "https://www.blackflag.dev/",
+    "size": "large",
```
Comment (shairez) [inline packages/docs/scripts/pages.json:75]: please reduce this to 2 props - href and size
thanks!

Comment (zanettin) [reply packages/docs/scripts/pages.json:75]: imo it should be `href` and `tag`, right?
@openblackflag do you have a minute to adjust that so we can merge the PR?

**Pull request merged**